### ISSUE TITLE
"Token validation failed" error when re-deploying

### ISSUE THREAD

**Issue by Nael-Pavegen** [mindsphere/mindconnect-nodejs#82]
Hi,

I can send data to my asset on Mindsphere with no problems through the mindconnect node, but as soon as I re-deploy the project on Node-RED, the mindconnect tries to onboard the agent again then fails with the message
`Error: Network error occured Unauthorized {"error":"token_validation_failed"`

So for continuous development and testing, I have to offboard the agent and create a new config for it then re-initiate the mindconnect node with a new deployment.

I was wondering if I can force the mindconnect node to onboard only once regardless of redeployment?

**Comment by sn0wcat** [comment]
We will not wait to 3.7.0 to fix this, it will be released in a bugfix release 3.6.1

**Comment by sn0wcat** [comment]
Fixed in 3.6.1
```bash
npm install @mindconnect/mindconnect-nodejs  --save should fix the problem
```
